docs/geba_network.md

```diff
@@ -0,0 +1,30 @@
+# GEBA
+
+The Global Energy Balance Archive (GEBA) is a database for the central storage of the worldwide measured energy fluxes at the Earth’s surface, maintained at ETH Zurich (Switzerland). Many of the historical records extend over several decades. GEBA contains monthly data from a variety of sources, namely from the World Radiation Data Centre (WRDC) in St.Petersburg, national weather services, different research networks (BSRN, ARM, SURFRAD), peer-reviewed publications, project and data reports, and personal communications. GEBA is publicly accessible [online](http://www.geba.ethz.ch). Supplementary data are also available [here](https://doi.org/10.1594/PANGAEA.873078). An overview of the measurement locations is presented in the figure below.
+
+```{figure} /graphics/geba.png
+:height: 400px
+:alt: stations in the GEBA network
+
+Distribution of the 2500 locations with observational data contained in GEBA. Red symbols indicate locations with at least one monthly entry in GEBA; yellow symbols signify locations with multiyear records (at least three years of data) [source](https://doi.org/10.5194/essd-9-601-2017).
```

Comment:
Is there a legend for this figure? Can we easily make this figure ourselves using Cartopy (requires that there's a list of station latitudes and longitudes)?

Reply:
There is no legend, this is how it is in the paper they made. They do provide the latitudes and longitudes, so we could potentially make it ourselves; however, they have them in collapsed categories, so data scraping might be a bit of a pain (but feel free to try if you want to...) :-)